server/auth/index.ts

```diff
@@ -0,0 +1,23 @@
+import express from 'express';
+import Setup from './google/passport';
+import googleRouter from './google';
+
+Setup();
+
+const router = express.Router();
+
+router.use(
+  '/is-auth', // add { credentials: 'include' } into fetch request for this endpoint to work
```

Comment:
would prefer this to be spelled, or at least add 'ed',  out for clarity
```suggestion
  '/is-authed', // add { credentials: 'include' } into fetch request for this endpoint to work
```

Reply:
I'll take care of this locally, then push so I can update the call as well